blueocean-core-js/src/js/UrlBuilder.js

```diff
@@ -77,9 +77,7 @@ export const buildRunDetailsUrl = (run) => {
         throw new Error('Could not extract URI components');
     }
 
-    return `/organizations/${organizationName}` +
-        `/${encodeURIComponent(fullName)}/detail` +
-        `/${detailName}/${runId}/pipeline`;
+    return `/organizations/${organizationName}` + `/${encodeURIComponent(fullName)}/detail` + `/${detailName}/${runId}/pipeline`;
```

Comment:
this one looks odd, but then should all be one string anyway probably

Reply:
```
return `/organizations/${organizationName/${encodeURIComponent(fullName)}/detai/${detailName}/${runId}/pipeline`;
```

Is how it should be, but it had been bad to start with, so IMO the formatting is not to be blamed on the tool

Reply:
agree with @scherler on this, if we use 160 lines, it should be a really long string